k8s/ubuntu/gsvineyard.Dockerfile

```diff
@@ -26,12 +26,12 @@ RUN cd /tmp && \
     cd /tmp/libvineyard && \
     python3 setup.py bdist_wheel && \
     cd dist && \
-    mkdir -p /opt/graphscope/dist && \
-    cp -f ./*.whl /opt/graphscope/dist && \
+    mkdir -p /opt/vineyard/dist && \
+    cp -f ./*.whl /opt/vineyard/dist && \
     pip3 install ./*.whl && \
     cd /tmp/libvineyard/modules/io && \
     python3 setup.py bdist_wheel && \
-    cp -f dist/* /opt/graphscope/dist && \
+    cp -f dist/* /opt/vineyard/dist && \
     pip3 install dist/* && \
```

Comment:
I have another question, since we already installed the wheels in dist in vineyard image, why do we need to install it again in graphscope image.